db/warehouse/migrate/20240821180638_update_external_form_object_key_index.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateExternalFormObjectKeyIndex < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :hmis_form_definitions, name: "index_hmis_form_definitions_on_external_form_object_key"
```

Comment:
this is fine, maybe should use the key to allow this to rollback if needed (not tested)
```suggestion
    remove_index :hmis_form_definitions, [:external_form_object_key]
```

Reply:
Thanks, good idea. Updated and tested reversability